skimage/measure/_regionprops.py

```diff
@@ -835,6 +846,114 @@ def regionprops(label_image, intensity_image=None, cache=True):
+    ...                    [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0],
+    ...                    [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0],
+    ...                    [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0],
+    ...                    [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0],
+    ...                    [0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0],
+    ...                    [1, 0, 1, 0, 0, 1, 1, 0, 1, 1, 0, 0, 1, 1, 1, 1, 1, 0],
+    ...                    [0, 1, 1, 1, 1, 1, 1, 1, 0, 1, 1, 0, 0, 0, 1, 1, 1, 1],
+    ...                    [0, 1, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1]])
+    >>> euler_number(SAMPLE)  # doctest: +ELLIPSIS
+    0...
+    >>> euler_number(SAMPLE, neighbourhood=4)  # doctest: +ELLIPSIS
+    2...
+    """
+
+    # as image can be a label image, transform it to binary
+    image = (image > 0).astype(np.int)
+    image = pad(image, ((1, 1),), mode='constant')
+
+    if np.ndim(image) == 2:
+
+        F = np.array([[0, 0, 0], [0, 1, 4], [0, 2, 8]])
+        if neighbourhood == 4:
+            coefs = [0,  1,  0,  0,  0,  0,  0, -
+                     1,  0,  1,  0,  0,  0,  0,  0,  0]
+        else:
+            coefs = [0,  0,  0,  0,  0,  0, -1,
+                     0,  1,  0,  0,  0,  0,  0, -1,  0]
+        bins = 16
+    else:  # 3D images
+        F = np.array([[[0, 0, 0], [0, 0, 0], [0, 0, 0]],
+                      [[0, 0, 0], [0, 1, 4], [0, 2, 8]],
+                      [[0, 0, 0], [0, 16, 64], [0, 32, 128]]])
+        coefs26 = np.array([0, 1, 1, 0, 1, 0, -2, -1, 1, -2, 0, -1, 0, -1, -1, 0,
+                            1, 0, -2, -1, -2, -1, -1, -2, -6, -3, -3, -2, -3, -2, 0, -1,
+                            1, -2, 0, -1, -6, -3, -3, -2, -2, -1, -1, -2, -3, 0, -2, -1,
+                            0, -1, -1, 0, -3, -2, 0, -1, -3, 0, -2, -1, 0, 1, 1, 0,
+                            1, -2, -6, -3, 0, -1, -3, -2, -2, -1, -3, 0, -1, -2, -2, -1,
+                            0, -1, -3, -2, -1, 0, 0, -1, -3, 0, 0, 1, -2, -1, 1, 0,
+                            -2, -1, -3, 0, -3, 0, 0, 1, -1, 4, 0, 3, 0, 3, 1, 2,
+                            -1, -2, -2, -1, -2, -1, 1, 0, 0, 3, 1, 2, 1, 2, 2, 1,
+                            1, -6, -2, -3, -2, -3, -1, 0, 0, -3, -1, -2, -1, -2, -2, -1,
+                            -2, -3, -1, 0, -1, 0, 4, 3, -3, 0, 0, 1, 0, 1, 3, 2,
+                            0, -3, -1, -2, -3, 0, 0, 1, -1, 0, 0, -1, -2, 1, -1, 0,
+                            -1, -2, -2, -1, 0, 1, 3, 2, -2, 1, -1, 0, 1, 2, 2, 1,
+                            0, -3, -3, 0, -1, -2, 0, 1, -1, 0, -2, 1, 0, -1, -1, 0,
+                            -1, -2, 0, 1, -2, -1, 3, 2, -2, 1, 1, 2, -1, 0, 2, 1,
+                            -1, 0, -2, 1, -2, 1, 1, 2, -2, 3, -1, 2, -1, 2, 0, 1,
+                            0, -1, -1, 0, -1, 0, 2, 1, -1, 2, 0, 1, 0, 1, 1, 0, ])
+
+        if neighbourhood == 6:
+            coefs = np.flip(coefs26)
```

Comment:
```suggestion
            coefs = np.flip(coefs26, tuple(range(coefs26.ndim)))
```

Reply:
I don't understand your suggestion @hmaarrfk ...

Reply:
The ability to flip the whole array was only introduced with numpy 1.15

We support at least 1.14 (1.13 for now, but I would have made the PR for 1.14 if it would have helped)